openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/membership/membership-details-dialog/membership-details-dialog.component.spec.ts

```diff
@@ -17,52 +17,38 @@ import { SubscriptionAccount } from '../subscription-account-interface';
 import { EnrollmentItem } from '../enrollment-item-interface';
 import { Plan } from '../plan-interface';
 import { MockComponent } from 'ng-mocks';
-import { MatTab, MatTabGroup } from '@angular/material/tabs';
 import { IconComponent } from '../../../shared/components/icon/icon.component';
 import { ContentCardComponent } from '../../../shared/components/content-card/content-card.component';
 import { DialogHeaderComponent } from '../../../shared/screen-parts/dialog-header/dialog-header.component';
 import { PrimaryButtonComponent } from '../../../shared/components/primary-button/primary-button.component';
 import { SecondaryButtonComponent } from '../../../shared/components/secondary-button/secondary-button.component';
 import { EnrollmentLineItemComponent } from '../../../shared/screen-parts/enrollment-line-item/enrollment-line-item.component';
-import { MatCard } from '@angular/material/card';
 import { ProgramPlanDetailsComponent } from '../../../shared/screen-parts/program-plan-details/program-plan-details.component';
+import { TabbedContentCardComponent } from '../../../shared/components/tabbed-content-card/tabbed-content-card.component';
+import { ITab } from '../../../shared/components/tabbed-content-card/tab.interface';
+import { By, element } from 'protractor';
 
 class MockActionService { }
 class MockMatDialog { }
-class MockElectronService { }
 
-describe('LinkedCustomerMembershipState', () => {
+fdescribe('LinkedCustomerMembershipState', () => {
```

Comment:
This should be a lint rule. If we don't have it turned on, we should do that
```suggestion
describe('LinkedCustomerMembershipState', () => {
```

Reply:
good catch didn't realize I left that in there